src/baseConfig.js

```diff
@@ -11,6 +11,8 @@ export class BaseConfig {
 
   constructor() {
     this._current = {
+      endpoint: null, // Endpoint to use for auth requests (local only). Null uses HttpClient.
```

Comment:
i got a bit confused when looking up _current.client. can it be added here to make it easier to find?
like client: null, // the currently used client. do not set it yourself as it's get set during configuration /see index.js)
or alike